ddb_cp.module

```diff
@@ -125,6 +125,8 @@ function ddb_cp_get_test_result($site = NULL, $number = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @TODO: This function never returns any XML data only page not found error code?
```

Comment:
This *function* does not return XML. [`_ddb_cp_save_test_result()`](https://github.com/cableman/ddb_cp/blob/master/ddb_cp.module#L280) prints the XML testresult. The XML is never returned.